pipelines/containerfiles/Containerfile.openvino.mlserver.mlflow

```diff
@@ -17,12 +17,16 @@ RUN mkdir /models && chown ovms:ovms /models
 
 # CHANGE THIS LINE TO MATCH YOUR MODEL
 COPY --chown=ovms:ovms $MODEL_DIR /models/1
-# Not deleting this file causes a bug
-RUN if echo "${MODEL_DIR}" | grep -q "tf2model"; then rm -f /models/1/fingerprint.pb; fi
 
-RUN chmod o+rwX /models/1
-# https://docs.openshift.com/container-platform/4.13/openshift_images/create-images.html#use-uid_create-images
-RUN chgrp -R 0 /models/1 && chmod -R g=u /models/1
+RUN <<EOF
```

Comment:
@davidesalerno in my understanding, every RUN instruction in a Dockerfile results in an additional layer in the resulting image. Sometimes there are good reasons to split RUN instructions into separate layers, but if there isn't, I think it's a better practice to keep them in a single layer. This can avoid size bloat in cases where, for example, two commands get run and the second removes some temporary files that the first one creates. While that example doesn't apply to this particular case right now, I think it's a reasonable idea to group these commands into a single RUN, so that it might be more obvious to people making changes in the future, that they can just put more commands into it too, if it makes sense.

Counter question, to make sure I'm not missing anything: are there reasons that you can think of that we _shouldn't_ group these commands into a single RUN like this?

Also, with reference to the "rearrange" part of your question: they're not rearranged into a different order, they will all be executed in the same order that they were in previously.

Reply:
Nice, didn't even know this is possible.